Fabric.Identity.API/scripts/Install-Identity-Utilities.psm1

```diff
@@ -26,6 +26,8 @@ else {
     Import-Module -Name $dosInstallUtilities.FullName
 }
 
+$ErrorActionPreference = "Stop"
```

Comment:
By setting this, the default action on any error is going to be stop...which I think is largely fine...